### ISSUE TITLE
Can't place tile on covering when grid on the planet

### ISSUE THREAD

**Issue by NotSoDana** [space-wizards/space-station-14#32046]
## Description

title

**Reproduction**

1. Create planet
2. Spawn or tp grid on it
3. Place tile
4. Use crowbar on it
5. Pick up tile item
6. Try to insert tile back

**Screenshots**

https://github.com/user-attachments/assets/d7fe92cc-82e7-47ec-8db9-53b012048244

**Comment by lzk228** [comment]
plating, not covering

**Comment by Killerqu00** [comment]
Could you please check if that grid has ProtectedGridComponent? This might be the reason why.

**Comment by NotSoDana** [comment]
> Could you please check if that grid has ProtectedGridComponent? This might be the reason why.

Nope, it hasn't. For more clarification, on the video above a planetary station grid implemented through a biome in the station prototype. But I also have checked how it works with a grid that was spawned on planet and the issue is the same. So tiles are not placed in any case if the grid is above the planet.
